script/build-newsletter-editions.js

```diff
@@ -96,14 +96,16 @@ async function saveNewsletterEditions(organizationId, letterheadData) {
   for await (let newsletter of letterheadData.items) {
     let headline = shared.cleanContent(newsletter.title);
 
-    if (!newsletter.publicationDate) {
+    if (newsletter.publicationStatus != 1) {
       //from Letterhead: Publication status of results to include (1 = published; 2 = scheduled; 4 = review; 6 = all statuses)
       console.log(
         '> Org#' +
           organizationId +
           ' Newsletter ID#' +
           newsletter.id +
           " '" +
+          newsletter.publicationStatus +
```

Comment:
This looks identical to what [main](https://github.com/news-catalyst/next-tinynewsdemo/blob/main/script/build-newsletter-editions.js#L107) already has.